std_msgs/Header header
uint32 frame_number
vicon_receiver/MarkerPosition[] markers